src/Responses/GetRecordingsResponse.php

```diff
@@ -27,23 +27,20 @@
  */
 class GetRecordingsResponse extends BaseResponse
 {
-    /**
-     * @var Record[]
-     */
-    private $records;
 
     /**
      * @return Record[]
      */
-    public function getRecords()
+    public function getRecords(): array
     {
-        if (null === $this->records) {
-            $this->records = [];
-            foreach ($this->rawXml->recordings->children() as $recordXml) {
-                $this->records[] = new Record($recordXml);
+        $records = [];
```

Comment:
Trade-off between more memory (old version) and more cpu (new version). Need a good argument to decided which solution to choose for the community. We assume the XML content is static and never changes in a single API call.